src/main/js/types/MergeStrategyInfo.ts

```diff
@@ -26,5 +26,5 @@ export interface MergeStrategyInfo {
   commitMessageDisabled: boolean;
   defaultCommitMessage: string;
   commitMessageHint: string;
-  commitAuthor: string;
+  commitAuthor: string | null;
```

Comment:
Is this actually null or undefined ? Then it could be an optional property, which I would personally prefer